make/migrations/postgresql/0011_1.10.0_schema.up.sql

```diff
@@ -0,0 +1,35 @@
+/*Table for keeping the plug scanner registration*/
+CREATE TABLE scanner_registration
+(
+    id SERIAL PRIMARY KEY NOT NULL,
+    uid VARCHAR(64) UNIQUE NOT NULL,
```

Comment:
Can we use the native uuid type of postgres here?

Reply:
No, decouple with the database driver.